### ISSUE TITLE
Default home search bar missing label

### ISSUE THREAD

**Issue by BuddyinKC** [omeka/omeka-s#2534]
The search bar on the default homepage (ours is https://exhibits.library.umkc.edu/) lacks a label or aria-label so screen readers don't announce the search bar properly.

**Comment by kimisgold** [comment]
@aviannamiller The fulltext search form has been updated to have `role="search"` on the form and an `aria-label` on the text input. Can you test to see if the form is sufficiently labeled for screen readers? These changes are on the `fix/minimal-search-label` branch.

**Comment by aviannamiller** [comment]
@kimisgold - Yes, that's working well and should be good to go.

**Comment by kimisgold** [comment]
@BuddyinKC This fix will appear in the next bugfix release.
